mock/mock.go

```diff
@@ -42,6 +42,9 @@ type Call struct {
 	// this method is called.
 	ReturnArguments Arguments
 
+	// Holds the caller info for the On() call
+	CallerInfo []string
```

Comment:
Would you mind unexporting this?

This package already exports too much of it's internals :)

Reply:
haha sure, I thought I'd follow the trend xD